docs/docs-content/vm-management/vm-migration-assistant/migrate-vms-vmo-cluster.md

```diff
@@ -0,0 +1,929 @@
+         }
+
+         # Create or clear log file
+         if (Test-Path $LogFile) {
+             Clear-Content -Path $LogFile
+         } else {
+             New-Item -Path $LogFile -ItemType File -Force
+         }
+
+         # Check for VirtIO Drivers
+         Write-Log "Checking for VirtIO Drivers..."
+         $VirtIODrivers = Get-WmiObject -Class Win32_PnPSignedDriver | Where-Object {
+             $_.DeviceName -like "*VirtIO*" -or $_.DriverProviderName -like "*Red Hat*"
+         }
+         if ($VirtIODrivers) {
+             Write-Log "VirtIO Drivers are installed."
+         } else {
+             Write-Log "VirtIO Drivers are NOT installed."
+         }
+
+         # Check for QEMU Agent
+         Write-Log "Checking for QEMU Guest Agent..."
+         $QemuAgentInstalled = Get-WmiObject -Class Win32_Product | Where-Object {
+             $_.Name -like "*QEMU*"
+         }
+         if ($QemuAgentInstalled) {
+             Write-Log "QEMU Guest Agent is installed."
+         } else {
+             Write-Log "QEMU Guest Agent is NOT installed."
+         }
+
+         # Check for VMware Tools (vmtoolsd)
+         Write-Log "Checking for VMware Tools (vmtoolsd)..."
+         $VmToolsInstalled = Get-Command -Name "vmtoolsd.exe" -ErrorAction SilentlyContinue
+         if ($VmToolsInstalled) {
+             Write-Log "VMware Tools (vmtoolsd) is installed."
+         } else {
+             Write-Log "VMware Tools (vmtoolsd) is NOT installed."
+         }
+
+         # Verbose Output
+         if ($VerboseOutput) {
+             Get-Content -Path $LogFile
+         }
+
+         Write-Log "Check completed. Log saved to $LogFile."
+     ```
+
+  2. Make any changes that you need, but ensure that `metadata.name` is set to `forklift-virt-customize`, and `metadata.namespace` is set to your target VM migration namespace.
+
+     Format rules:
+
+     - Windows scripts must follow the regex `^([0-9]+_win_firstboot(([\w\-]*).ps1))$`.
+       - For example, `00_win_firstboot.ps1` is a PowerShell script that runs at boot.
+     - Linux scripts must follow the regex `^([0-9]+_linux_(run|firstboot)(([\w\-]*).sh))$`.
+       - `firstboot` means run at first boot.
+       - `run` means execute after virt-v2v conversion, but before the VM starts.
+     - The number at the beginning of the key determines the script execution order.
+
+  3. Ensure that your terminal is [configured to access your VMO cluster](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) and issue the following command to create the ConfigMap on your cluster. 
+
+     ```shell
+     kubectl apply -f configmap.yaml
+     ```
+
+  </details>
+
+## Migrate VMs
+
+### Create Source Provider
+
+1. Log in to the VM Migration Assistant.
+
+2. From the left **Main Menu**, select **Providers for virtualization**.
+
+3. In the top-left corner, use the **Namespace** drop-down Menu to select your Kubernetes namespace for the migration.
```

Comment:
It's unclear what this namespace is used for?

Reply:
@cloudmaniac When you create a plan or provider, they may not show unless you have selected the correct namespace (or All namespaces).